### PR TITLE
update whisper-create with all options

### PR DESCRIPTION
**Pull request by AnderEnder**

### CODE REVIEW

File: whisper/src/bin/whisper-create.rs [hunk 1]
```diff
@@ -55,58 +56,74 @@ struct Args {
...
-    let file = WhisperBuilder::default()
-        .add_retentions(&args.retentions)
-        .x_files_factor(args.x_files_factor)
-        .aggregation_method(args.aggregation_method)
-        .sparse(args.sparse)
-        .build(args.path)?;
-
-    println!("{:#?}", file.info());
+    if args.estimate {
+        for (i, retention) in args.retentions.iter().enumerate() {
```
Comment (loony-bean) [inline whisper/src/bin/whisper-create.rs]: Consider moving into separate fn

File: whisper/src/bin/whisper-create.rs [hunk 2]
```diff
@@ -55,58 +56,74 @@ struct Args {
...
-
-// whisper-create.py load.1m.wsp 60:1440 120:2880
-// [ERROR] File load.1m.wsp already exists!
-
-// whisper-create.py load.2m.wsp 60:1440 120:2880
-// Created: load.2m.wsp (51880 bytes)
-
-// whisper-create.py load.2m.wsp 60:1440 120:1440
-// Created: load.2m.wsp (34600 bytes)
+fn byte_format(number: usize) -> String {
```
Comment (loony-bean) [inline whisper/src/bin/whisper-create.rs]: crate human_size?

Comment (loony-bean) [reply whisper/src/bin/whisper-create.rs]: humansize, pardon

File: whisper/src/bin/whisper-create.rs [hunk 3]
```diff
@@ -55,58 +56,78 @@ struct Args {
...
-        .build(args.path)?;
-
-    println!("{:#?}", file.info());
+    if args.estimate {
+        estimate_info(&args.retentions);
+    } else {
+        if args.overwrite && args.path.exists() {
+            println!(
+                "Overwriting existing file: {}",
+                &args.path.to_str().unwrap()
```
Comment (andy128k) [inline whisper/src/bin/whisper-create.rs]: ?

File: whisper/src/bin/whisper-create.rs [hunk 4]
```diff
@@ -55,58 +56,78 @@ struct Args {
...
+
+        WhisperBuilder::default()
+            .add_retentions(&args.retentions)
+            .x_files_factor(args.x_files_factor)
+            .aggregation_method(args.aggregation_method)
+            .sparse(args.sparse)
+            .build(&args.path)?;
+
+        let size = args.path.metadata()?.len();
+        println!("Created: {} ({} bytes)", &args.path.to_str().unwrap(), size);
```
Comment (andy128k) [inline whisper/src/bin/whisper-create.rs]: ?

**Pull request merged**